api/arceos_posix_api/src/lib.rs

```diff
@@ -0,0 +1,58 @@
+#![cfg_attr(all(not(test), not(doc)), no_std)]
+#![feature(ip_in_core)]
```

Comment:
Add `#![feature(doc_cfg)]` and `#![feature(doc_auto_cfg)]` to generate features info in doc.